app/services/jira_board_service.rb

```diff
@@ -0,0 +1,91 @@
+class JiraBoardService
+  attr_accessor :jira_board
+
+  def update!(params)
+    @jira_board = JiraBoard.find_by(id: params['id'])
+    jira_board.update!(update_params(params))
+    update_environments(params)
+    jira_board
+  end
+
+  def create!(params)
```

Comment:
I'd suggest splitting this class into two. One for creating, and the other for updating.